### DISCUSSION TITLE
Change value of work time entry after a period has been approved

### DISCUSSION THREAD

**Discussion by klausjettkant** [kimai/kimai#4953] [Questions & Answers]
When I am billing my costumers, I use time-sheet exports to prove the delivery of accomplished activities. All the work that has been done is written down in the description field of a work time entry and it is associated to a certain project. If I want to correct spellings errors or change the work to antoher project I get the error message: "This period has already been approved, you cannot edit or create new entries before 01.07.24." Managing projeject I have to change theese values via database kimai2_timesheets quit often, but it is a tremendoues effort in my workflow. Will there be a change to this behaviour? Until that change, am I able to deactivate the write protection? Thank you in advance.

**Comment by kevinpapst** [comment]
> but it is a tremendoues effort in my workflow

Lock the month after your invoices... ?

**Comment by klausjettkant** [comment]
Thank you for your suggestion. Projects may be active for up to half a year. Does it affect the flextime wage record of the employees when I do not lock the months? Thank you in advance.
